### DISCUSSION TITLE
Using HTML in Calendar description

### DISCUSSION THREAD

**Discussion by svicino** [add2cal/add-to-calendar-button#274] [Q&A] [answered]
Is there any way to use HTML in the calendar description? We have some bold text and list elements that would look much better styled as HTML.

**Comment by jekuer** [comment, marked as answer]
Yes, it is - somehow.


From the [documentation](https://github.com/add2cal/add-to-calendar-button/blob/main/DOCS.md):
> In case you want to customize them, you can alternatively set the option customLabels and provide a JSON object where the key would be the text block's identifier (lower case without spaces). See the bottom of the js file for possible text blocks. You can use HTML pseudo tags, which get transformed automatically (but not for styled labels). Suported ones: [url] (see next point), [br], [hr], [p], [strong], [u], [i], [em], [li], [ul], [ol], [h*] (like h1, h2, h3, ...).
Formatting a URL in the description like [url]https://....[/url] makes it clickable. [url]https://....|URL Text[/url] defines a linked textblock saying "URL Text" (not supported by Apple, iCal, and Yahoo; not supporting special characters).
